examples/3d/deferred_rendering.rs

```diff
@@ -193,7 +193,7 @@ fn setup(
         PbrBundle {
             mesh: meshes.add(Cuboid::new(2.0, 1.0, 1.0)),
             material: materials.add(StandardMaterial {
-                base_color: LegacyColor::hex("888888").unwrap(),
+                base_color: Srgba::hex("888888").unwrap().into(),
```

Comment:
```suggestion
                base_color: Color::hsl(0., 0., 0.533),
```

No need to have that gross `unwrap`